roles/sro_deploy_art_bundle/tasks/main.yml

```diff
@@ -0,0 +1,245 @@
+---
+- name: Add brew pull secrets
+  block:
+    - name: Grab the current secrets
+      shell:
+        oc get secret/pull-secret -n openshift-config -o json
+        | jq -r '.data.".dockerconfigjson"'
+        | base64 -d
+        > "{{ artifact_extra_logs_dir }}/authfile"
```

Comment:
@pacevedom you shouldn't store `authfile` and `dockerconfigjson` files in the `extra_logs_dir`, both of them have secret content!! 
you can thank the testplatform team for doing the proper censoring that prevented the leak here ;-)

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/247/pull-ci-openshift-psap-ci-artifacts-master-sro-e2e-master-art/1438537855172349952/artifacts/e2e-master-art/nightly/artifacts/006__sro__deploy_art_bundle/authfile
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/247/pull-ci-openshift-psap-ci-artifacts-master-sro-e2e-master-art/1438537855172349952/artifacts/e2e-master-art/nightly/artifacts/006__sro__deploy_art_bundle/dockerconfigjson

I guess you can write into `/tmp` *and* delete the files in a `always` block!